### PR TITLE
Fixes ontology lta_task update (bug: #162)

### PR DESCRIPTION
**Pull request by oplatek**
See https://github.com/UFAL-DSG/alex/issues/162 for description.
Tests are passing. 
I have not created the corresponding tests (on my TODO list, I want to ask @ticcky  how it should be tested).
The fix assumes that ``from_stop`` and to ``to_stop`` slots will trigger only ``find_connection`` task.

### CODE REVIEW

File: alex/applications/PublicTransportInfoCS/data/ontology.py [hunk 1]
```diff
@@ -302,7 +302,8 @@
         'lta_task': {
             'weather': [('', '^task$', '^weather$'), ],
             'find_connection': [('', '^task$', '^find_connection$'), ('', '^departure_', ''), ('', '^arrival_', ''),
-                                ('', '^duration$', '')],
+                                ('', '^duration$', ''), 
+                                ('', '^from_stop$', ''), ('', '^to_stop$', ''),],
```
Comment (ticcky) [inline alex/applications/PublicTransportInfoCS/data/ontology.py]: This will not play well with the `find_platform` task below, as the last-talked-about inference will switch to the `find_connection` task whenever the user mentions `from_stop` or `to_stop`.

File: alex/applications/PublicTransportInfoCS/hdc_policy.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def __init__(self, cfg, ontology):
         self.system_das = []
         self.last_system_dialogue_act = None
 
-        self.debug = cfg.getpath('DM/basic/debug', False)
+        self.debug = cfg['DM']['basic']['debug']
```
Comment (ticcky) [inline alex/applications/PublicTransportInfoCS/hdc_policy.py]: Why did you change this back?

**Comment by ticcky** [comment]
I pointed out the problem I see with your fix in the code. I think a way how to fix this is to make SLU produce `inform(task=find_connection)` for the hypothesis `<hypothesis p="0.720">JÁ BYCH CHTĚL JET _EHM_HMM_ Z MALOSTRANSKÉHO NÁMĚSTÍ NA OŘECHOVKU</hypothesis>`.

**Comment by tuetschek** [comment]
You are right, this will probably fix it. It's actually strange that this is not yet recognized as ``inform(task=find_connection)``.

You could also want to say just ``Z MALOSTRANSKÉHO NÁMĚSTÍ NA OŘECHOVKU``, which Lukáš's suggestion will not fix, but we could consider it ambiguous...

**Comment by oplatek** [comment]
OK, I will close the PR, I agree that SLU should detect the ``inform(task=find_connection)`` which should find it.
